packages/widget/src/components/transfer-status/transfer-status.ts

```diff
@@ -0,0 +1,57 @@
+import { html, type HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { BaseComponent } from '../base-component/base-component';
+import { greenMark, networkIconsMap } from '../../assets';
+import { styles } from './styles';
+
+@customElement('sygma-transfer-status')
+export class TransferStatus extends BaseComponent {
+  static styles = styles;
+
+  @property({ type: String }) from: string = '';
+
+  @property({ type: String }) to: string = '';
+
+  @property({ type: String }) amount: string = '';
```

Comment:
```suggestion
  @property({ type: Number }) amount: number = '';
```

Reply:
why not string since probably we are going to format the amount? 🤔

Reply:
why would parent component be responsible of how transfer component will format amount? :)